### DISCUSSION TITLE
Some pesky cookies refuses to get deleted

### DISCUSSION THREAD

**Discussion by user2026** [Cookie-AutoDelete/Cookie-AutoDelete#1173] [Q&A / Support] [answered]
Edit: I, of course, looked at the other discussions after i made this post. And i guess i got the answer there. I don't see a way to delete this

This is annoying me more than it should. But there is no obvious reason for the cookies to remain. 
When i browse reddit and there are twitch clips in the feed, i get 5 cookies from twitch.tv, after i close the tab, CAD log says they were deleted, but when checking "manage data" they are still there. 

I use firefox with Multi-Account Container and Temporary Containers installed, can it be some sort of conflict with those extensions?

**Comment by kennethtran93** [comment]
What is your privacy setting at?

Multi account shouldn't be a problem (I use it too). Temporary containers shouldn't be an issue if they are destroyed not too long after as it should remove the cookies then.

**Reply by user2026** [reply]
It was at Strict. But i changed it to custom All third-party cookies. Seems like the cookies gets retained for some reason when using Strict. But from what i can tell, it only happens with the twitch cookies

**Reply by kennethtran93** [reply]
Yeah unfortunately strict is wrecking havoc on webextensions using cookie api. Until somebody who understands the underlying code and can add in the functions needed for the new values, we have to make do without the new cookie protection if you want cad to function properly.

Note that the setting changes may not affect existing cookies - those will need to be manually removed either by deleting the container or through site data.

**Reply by user2026** [reply]
Okay, thanks for the explanation

**Comment by kennethtran93** [comment, marked as answer]
Current solution is to not use strict mode / cross-site cookie protection. Standard mode, or custom mode with Third-party tracking cookie option still works and was the previous default for strict pre 86.
